src/components/navbar.svelte

```diff
@@ -124,19 +126,18 @@
     <Menu bind:this={options}>
       <List>
         <Item on:SMUI:action={option1}><Text>About</Text></Item>
-        <Item on:SMUI:action={() => option('Like Unified Arcade App?', 0)}><Text>Like UAA?</Text></Item>
-        <Item on:SMUI:action={() => option('Would you like to buy the licensed version?', 1)}><Text>Purchase</Text></Item>
-        <Item on:SMUI:action={() => option('Want another icon to show up?', 2)}><Text>another icon?</Text></Item>
+        <Item on:SMUI:action={() => option('Like Unified Arcade App?', 0, 'Like UAA')}><Text>Like UAA?</Text></Item>
+        <Item on:SMUI:action={() => option('Would you like to buy the licensed version?', 1, 'Purchase')}><Text>Purchase</Text></Item>
+        <Item on:SMUI:action={() => option('Want another icon to show up?', 2, 'Icon')}><Text>another icon?</Text></Item>
```

Comment:
I could have also put the content and titles in an array or object like, [uaa, Purchase....] and passed the index, but the titles are not too long so I am content with this.

Reply:
nah, this is fine dude.